app/components/work_packages/split_view_component.sass

```diff
@@ -1,6 +1,6 @@
 .op-work-package-split-view
   height: 100%
-  width: 550px
+  min-width: 550px
```

Comment:
❓ You should be able to use the variable here as well, right?

Reply:
No, as the split screen is designed to shrink below the variable value if the screen size is not large enough to fit everything in it. However, it shall not shrink below the 550px.